src/components/MailList.tsx

```diff
@@ -173,15 +171,50 @@ export const MailList = ({
 };
 
 const EmptyComponent = () => (
-  <View
-    style={{
-      justifyContent: 'center',
-      alignItems: 'center',
-      paddingVertical: spacing.xxl,
-    }}>
+  <View style={styles.emptyComponentContainer}>
     <Icon name="file-tray-outline" size={50} color={colors.skyLight} />
     <Text style={[fonts.large.regular, { color: colors.skyLight }]}>
       {'No Messages'}
     </Text>
   </View>
 );
+
+const styles = StyleSheet.create({
```

Comment:
Could you please move styles to separate file, if needed create a folder for component